### PR TITLE
Add zypper service commands needed for registration

### PR DESCRIPTION
**Pull request by djoreilly**
Adds addService(), refreshService() and enableServiceAutorefresh()

### CODE REVIEW

**Comment by djoreilly** [comment]
need to add install_release_package

**Review by skazi0 — APPROVED** [review]
lgtm

File: connect/zypper.go [hunk 1]
```diff
@@ -120,12 +142,48 @@ func removeService(serviceName string) error {
...
 	_, err := zypperRun(args, true, []int{zypperOK})
 	return err
 }
 
+func installReleasePackage(identifier string) error {
+	if identifier == "" {
+		return nil
+	}
+	args := []string{"rpm", "-q", identifier + "-release"}
+	if _, err := execute(args, false, nil); err != nil {
```
Comment (djoreilly) [inline connect/zypper.go:168]: this is wrong - it should return if there is no error, ie the rpm is already installed

Comment (djoreilly) [reply connect/zypper.go]: done

Comment (skazi0) [reply connect/zypper.go]: ah, right, the go-style error handling is very error prone wrt negative tests. maybe rename this to `ok` to make it clear what we're up to?

Comment (djoreilly) [reply connect/zypper.go]: no it was my mistake reading the ruby "unless" statement. I added a comment with the latest push

Comment (skazi0) [reply connect/zypper.go]: `unless` is very error prone as well :o)
people are not good at inverse logic.

**Review by skazi0 — APPROVED** [review]
lgtm

**Pull request merged**